files/zh-cn/web/javascript/reference/global_objects/globalthis/index.html

```diff
@@ -30,7 +30,7 @@ <h2 id="描述">描述</h2>
 
 <p>在以前，从不同的 JavaScript 环境中获取全局对象需要不同的语句。在 Web 中，可以通过 <code>window</code>、<code>self</code> 或者 <code>frames</code> 取到全局对象，但是在 <a href="/zh-CN/docs/Web/API/Worker">Web Workers</a> 中，只有 <code>self</code> 可以。在 Node.js 中，它们都无法获取，必须使用 <code>global</code>。</p>
 
-<p>在松散模式下，可以在函数中返回 <code>this</code> 来获取全局对象，但是在严格模式和模块环境下，<code>this</code> 会返回 <code>undefined</code>。 You can also use <code>Function('return this')()</code>, but environments that disable {{jsxref("eval", "eval()")}}, like <a href="/zh-CN/docs/Glossary/CSP">CSP</a> in browsers, prevent use of {{jsxref("Function")}} in this way.</p>
+<p>在松散模式下，可以在函数中返回 <code>this</code> 来获取全局对象，但是在严格模式和模块环境下，<code>this</code> 会返回 <code>undefined</code>。 你也可以使用 <code>Function('return this')()</code>，但是那些诸如浏览器中的<a href="/zh-CN/docs/Glossary/CSP">CSP</a>{{jsxref("eval", "eval()")}}，不容许我们这样使用{{jsxref("Function")}}。</p>
```

Comment:
> 但是那些诸如浏览器中的<a href="/zh-CN/docs/Glossary/CSP">CSP</a>{{jsxref("eval", "eval()")}}

Please improve you translation to present the meaning of "something like CSP in browser may lead to environment to disabled eval". Simply put `<a href="/zh-CN/docs/Glossary/CSP">CSP</a>{{jsxref("eval", "eval()")}}` together is nonsense.

Reply:
Sorry, I have revised the translation, please review it again, thanks.